packages/sdk/src/fee/feeHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { FeeHandlerRouter__factory } from '@buildwithsygma/sygma-contracts';
+import { ethers } from 'ethers';
+
+export const getFeeHandlerAddress = async (
+  signerOrProvider: ethers.providers.JsonRpcProvider | ethers.Signer,
+  feeRouterAddress: string,
+  domainId: string,
+  resourceId: string,
+): Promise<string | Error> => {
```

Comment:
It is connected with next comment
```suggestion
): Promise<string> => {
```